pkg/operator/ceph/csi/version.go

```diff
@@ -59,7 +53,7 @@ func (v *CephCSIVersion) String() string {
 
 // SupportsOMAPController checks if the detected version supports OMAP generator
 func (v *CephCSIVersion) SupportsOMAPController() bool {
```

Comment:
SupportsOMAPController has not required anymore we can remove this one. This was added because we were supporting cephcsi older versions also.

Reply:
sure, thought of keeping some checks in place. any way removed it completely .thanks..